extensions/wikia/FounderEmails/events/FounderEmailsViewsDigestEvent.class.php

```diff
@@ -5,24 +5,19 @@ public function __construct( Array $data = array() ) {
 		$this->setData( $data );
 	}
 
-	public function enabled ( $wgCityId, User $user ) {
+	public function enabled ( $wikiId, User $user ) {
 		if ( self::isAnswersWiki() ) {
 			return false;
 		}
 
-		// disable if all Wikia email disabled
-		if ( $user->getBoolOption( 'unsubscribed' ) ) {
-			return false;
-		}
-
 		// If complete digest mode is enabled, do not send views only digest
-		if ( $user->getOption( "founderemails-complete-digest-$wgCityId" ) ) {
+		if ( $user->getOption( "founderemails-complete-digest-$wikiId" ) ) {
 			return false;
 		}
-		if ( $user->getOption( "founderemails-views-digest-$wgCityId" ) ) {
+		if ( $user->getOption( "founderemails-views-digest-$wikiId" ) ) {
 			return true;
```

Comment:
It looks like you pulled these 2 checks into the `FounderPageViewsDigestController` but didn't remove them from here?